dm in cloud: updated docs as per new ui changes (#11966)
pingcap/docs: tidb-cloud/tidb-cloud-import-local-files.md

        >
        > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.

>     2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane. Alternatively, you can click **...** at the end of the cluster row and then click **Import**.

Reviewer: Click **...** at the end of the cluster row and then click **Import**.Alternatively, you can click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.

>     2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane. Alternatively, you can click **...** at the end of the cluster row and then click **Import**.

Author: ```suggestion
    2. Click **...** at the end of the cluster row and then click **Import**. Alternatively, you can click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
```

> 2. On the **Import** page, you can directly drag and drop your local file to the upload area to upload it, or click the upload area to select and upload the target local file. Note that the file name must have a ".csv" extension and must be no more than 50 MiB.

Reviewer: Note that  only one CSV file  and less than 50 MiB is supported

> 2. On the **Import** page, you can directly drag and drop your local file to the upload area to upload it, or click the upload area to select and upload the target local file. Note that the file name must have a ".csv" extension and must be no more than 50 MiB.

Author: ```suggestion
2. On the **Import** page, you can directly drag and drop your local file to the upload area to upload it, or click the upload area to select and upload the target local file. Note that only one CSV file of less than 50 MiB is supported.
```

4. Check the table.

    Here you can see a list of configurable table columns. Each line shows the table column name inferred by TiDB Cloud, the table column type inferred, and the previewed data from the CSV file.

    - If you import data into an existing table in TiDB Cloud, the column list is extracted from the table definition, and the previewed data is mapped to the corresponding columns by column names.

    - If you want to create a new table, the column list is extracted from the CSV file, and the column type is inferred by TiDB Cloud. For example, if the previewed data is all integers, the inferred column type will be **int** (integer).

5. Configure the column names and data types.

    If the first row in the CSV file records the column names, make sure that **Use first row as column name** is selected, which is selected by default.

    If the CSV file does not have a row for the column names, do not select **Use first row as column name**. In this case:

    - If the CSV table already exists, make sure the order of the columns is the same as the column list of the target table.

>     - If the CSV table is not created yet, input the names for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.

Author: ```suggestion
    - If the target table is not created yet, input the names for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.
```

> 6. Set the primary key. You can select a column as the primary key, or select multiple columns to use them as the composite primary key. Note that the order you select the columns matters when you select multiple columns.

Reviewer: Please refer to the following  notes ：
Select multiple fields to form a joint primary key.
The joint primary key will be formed in the order of the selected field names.
The primary key of the table is clustered index and cannot be deleted after creation.
It is necessary to ensure that the data corresponding to the primary key field is not empty and not repeated, otherwise the data import will fail.

> 6. Set the primary key. You can select a column as the primary key, or select multiple columns to use them as the composite primary key. Note that the order you select the columns matters when you select multiple columns.

Author: ```suggestion
6. Set the primary key. You can select a column as the primary key, or select multiple columns to create a joint primary key. The joint primary key will be formed in the order of the selected column names.

    > **Note**
    >
    > - The primary key of the table is a clustered index and cannot be deleted after creation.
    > - Ensure that the data corresponding to the primary key field is unique and not empty, otherwise the import task will fail. 
```

> 6. Set the primary key. You can select a column as the primary key, or select multiple columns to use them as the composite primary key. Note that the order you select the columns matters when you select multiple columns.

Author: > joint primary key

好像用 composite primary key 的更多 https://www.google.com/search?q=joint+primary+key&ei=ygS8Y5y3AdmK-QaU94WgBA&ved=0ahUKEwicxM6Au7r8AhVZRd4KHZR7AUQQ4dUDCBA&uact=5&oq=joint+primary+key&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIFCAAQgAQyBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBQgAEIAEMgQIABAeMggIABAFEB4QCjIICAAQBRAeEAoyBggAEAUQHjILCAAQBRAeEPEEEAo6CQgAEAcQHhDxBEoECEEYAEoECEYYAFAAWJkHYIIWaABwAXgAgAH4AYgBkAeSAQUwLjQuMZgBAKABAcABAQ&sclient=gws-wiz-serp#ip=1

        >
        > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.

>     2. Click **...** at the end of the cluster row and then click **Import**.

Reviewer 2: According to PMs, the **Import** option will be removed from ... next week, to avoid updating the doc again, can we tell users to click the name of your target cluster to go to its overview page?

4. Check the table.

    Here you can see a list of configurable table columns. Each line shows the table column name inferred by TiDB Cloud, the table column type inferred, and the previewed data from the CSV file.

    - If you import data into an existing table in TiDB Cloud, the column list is extracted from the table definition, and the previewed data is mapped to the corresponding columns by column names.

    - If you want to create a new table, the column list is extracted from the CSV file, and the column type is inferred by TiDB Cloud. For example, if the previewed data is all integers, the inferred column type will be **int** (integer).

5. Configure the column names and data types.

    If the first row in the CSV file records the column names, make sure that **Use the first row as column name** is selected, which is selected by default.

    If the CSV file does not have a row for the column names, do not select **Use the first row as column name**. In this case:

    - If the target table already exists, make sure the order of the columns is the same as the column list of the target table.

>     - If the target table is not created yet, input the names for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.

Reviewer 2: ```suggestion
    - If you need TiDB Cloud to create the target table, input the names for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.
```

6. Set the primary key. You can select a column as the primary key, or select multiple columns to create a composite primary key. The composite primary key will be formed in the order of the selected column names.

    > **Note:**
    >
    > - The primary key of the table is a clustered index and cannot be deleted after creation.
    > - Ensure that the data corresponding to the primary key field is unique and not empty. Otherwise the import task will result in data inconsistency.

7. Edit the CSV configuration if needed.

   To edit the CSV configuration for more fine-grained control, you can also click **Edit CSV configuration**. For more information about the CSV configuration, see [CSV Configurations for Importing Data](/tidb-cloud/csv-config-for-import-data.md).

8. Click **Start Import**.

    You can view the import progress on the **Import Task Detail** page. If there are warnings or failed tasks, you can check to view the details and solve them.

> 9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Try Chat2Query](/tidb-cloud/tidb-cloud-quickstart.md#step-2-try-chat2query-beta).

Author: ```suggestion
9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).
```

3. In the **Target** area, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. Click **Next**.

4. Check the table.

    You can see a list of configurable table columns. Each line shows the table column name inferred by TiDB Cloud, the table column type inferred, and the previewed data from the CSV file.

    - If you import data into an existing table in TiDB Cloud, the column list is extracted from the table definition, and the previewed data is mapped to the corresponding columns by column names.

    - If you want to create a new table, the column list is extracted from the CSV file, and the column type is inferred by TiDB Cloud. For example, if the previewed data is all integers, the inferred column type will be **int** (integer).

> 5. Configure the column names and data types.

Reviewer 2: Is this step only applicable to the new target table?

    If the first row in the CSV file records the column names, make sure that **Use the first row as column name** is selected, which is selected by default.

    If the CSV file does not have a row for the column names, do not select **Use the first row as column name**. In this case:

    - If the target table already exists, make sure the order of the columns is the same as the column list of the target table.

>     - If the target table is not created yet, input the name for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.

Author: ```suggestion
    - If you need TiDB Cloud to create the target table, input the name for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.
```

> 6. Set the primary key. You can select a column as the primary key, or select multiple columns to create a composite primary key. The composite primary key will be formed in the order in which you select the column names.

Author: ```suggestion
6. For a new target table, you can set the primary key. You can select a column as the primary key, or select multiple columns to create a composite primary key. The composite primary key will be formed in the order in which you select the column names.
```

    > **Note:**
    >
    > - The primary key of the table is a clustered index and cannot be deleted after creation.

>     > - Ensure that the data corresponding to the primary key field is unique and not empty. Otherwise the import task will result in data inconsistency.

Reviewer 2: ```suggestion
    > - Ensure that the data corresponding to the primary key field is unique and not empty. Otherwise, the import task will result in data inconsistency.
```
